alacritty_terminal/src/window.rs

```diff
@@ -267,23 +269,29 @@ impl Window {
 
         let icon = Icon::from_bytes_with_format(WINDOW_ICON, ImageFormat::ICO);
 
-        WindowBuilder::new()
+        let mut builder = WindowBuilder::new()
             .with_title(title)
             .with_visibility(false)
             .with_transparency(true)
             .with_decorations(decorations)
             .with_maximized(window_config.startup_mode() == StartupMode::Maximized)
             .with_window_icon(icon.ok())
             // X11
-            .with_class(class.into(), DEFAULT_NAME.into())
+            .with_class(instance.into(), class.into())
             // Wayland
-            .with_app_id(class.into())
+            .with_app_id(instance.into());
+
+        if let Some(ref val) = window_config.gtk_theme_variant {
+            builder = builder.with_gtk_theme_variant(val.clone())
+        }
+        builder
```

Comment:
I'd personally prefer an empty line before `builder` too, but that's me being extremely pedantic. I do think that it would look nicer though.

Reply:
I forgot about that one.